sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -140,19 +140,23 @@ To find the resource ID:
 1. In the resulting JSON, copy the value of the `id` property.
 
 ```C# Snippet:QueryResource
-string resourceId = "/subscriptions/<subscription_id>/resourceGroups/<resource_group_name>/providers/<resource_provider>/<resource>";
-var client = new LogsQueryClient(new DefaultAzureCredential());
-
-Response<LogsQueryResult> result = await client.QueryResourceAsync(
-    new ResourceIdentifier(resourceId),
-    "AzureActivity | top 10 by TimeGenerated",
-    new QueryTimeRange(TimeSpan.FromDays(1)));
-
-LogsTable table = result.Value.Table;
-
-foreach (LogsTableRow row in table.Rows)
+string tableName = "<table_name>";
+var results = await client.QueryResourceAsync(
+    new ResourceIdentifier(TestEnvironment.WorkspacePrimaryResourceId),
```

Comment:
This can't be used in a sample because users won't know what a TestEnvironment is.